### PR TITLE
feat: steps

### PR DESCRIPTION
**Pull request by pearofducks**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -69,32 +69,34 @@ export const box = {
...
+  stepDot: 'rounded-full border-2 h-20 w-20 transition-colors duration-300',
   stepDotVertical: '',
   stepDotVerticalLeft: '',
   stepDotVerticalRight: 'col-start-2',
   stepDotHorizontal: 'row-start-2 justify-self-end',
 
-  stepDotActive: 'border-blue-600 bg-blue-600 text-blue-600',
-  stepDotComplete: 'border-blue-600 bg-blue-600 text-white',
-  stepDotIncomplete: 'border-bluegray-300 bg-white text-white',
+  stepDotActive: 'i-border-$color-step-indicator-handle-border-active i-bg-$color-step-indicator-handle-background-active i-text-$color-step-indicator-handle-background-active',
```
Comment (AnnaRybkina) [inline index.js:84]: Curious, we use the same token for both background and text. Is it intentional? Couldn't find any token for text in the Taxonomy sheet, is it missing or i it how it is supposed to be

File: index.js [hunk 2]
```diff
@@ -69,32 +69,34 @@ export const box = {
...
+  stepDotComplete: 'i-border-$color-step-indicator-handle-border-active i-bg-$color-step-indicator-handle-background-active i-text-$color-step-indicator-icon',
+  stepDotIncomplete: 'i-border-$color-step-indicator-handle-border i-bg-$color-step-indicator-handle-background i-text-$color-step-indicator-icon',
 
-  stepLine: 'step-line transition-colors duration-300',
+  stepLine: 'group-last/stepv:hidden step-line transition-colors duration-300',
```
Comment (AnnaRybkina) [inline index.js]: Can we remove `step-line` now? Checking [here](https://github.com/fabric-ds/css/blob/49b07c05686ef132b203026e941bf04dbab6fc7a/src/components/steps.css#L19) and after the `hidden` fix it is probably not needed? 
```suggestion
  stepLine: 'group-last/stepv:hidden transition-colors duration-300',
```

File: index.js (hunk 1 above)
Comment (pearofducks) [reply index.js:84]: Yep intentional! We want the checkbox to be 'invisible' in this state, so we just set it to the same color as the background.

File: index.js (hunk 2 above)
Comment (pearofducks) [reply index.js]: Good catch, thanks!

**Review by AnnaRybkina — APPROVED** [review]
Great stuff! Let’s ship

**Pull request merged**